app/addons/activetasks/assets/less/activetasks.less

```diff
@@ -63,3 +64,148 @@
   }
 }
 
+.no-matching-database-on-search {
+  color: #e33f3b;
+}
+
+p.multiline-active-tasks-message {
+  margin: 0;
+
+  &.time:nth-child(2) {
+    color: #888;
+  }
+}
+
+#dashboard-upper-content {
```

Comment:
documents contains a def for this also - but there don't seem to be side effects. It's minor anyway